.github/workflows/ci.yml

```diff
@@ -24,3 +24,9 @@ jobs:
       - name: Golang Tests
         run: go test -v ./...
         working-directory: ./Backend
+  frontend_testing:
```

Comment:
if we're calling this frontend_testing perhaps we should call the other one backend_testing